src/provider/cloud-element-templates/Validator.js

```diff
@@ -6,6 +6,8 @@ import {
 
 import semverCompare from 'semver-compare';
 
+import BpmnModdle from 'bpmn-moddle';
```

Comment:
I see this is only a _dev dependency_. Is this intended?

Reply:
It should be moved to a production dependency now